File: Synchronisation/MyDreamHouseSync/SynchronisationDataManager/Tables/HauspaketAttributWert.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static SynchronisationShared.SharedEnums;

namespace SynchronisationDataManager.Tables
{
    class HauspaketAttributWert : Table
    {
        public override string GetInsertString(DataRow row, DatabaseType type)
        {
            string query = "insert into hauspaket_attribut_wert (wert_id,attribut_id,wert_text,wert_ordnung,archived) values (";

            query = AddParam(query, row, 0, DataType.NUMERIC, false);
            query = AddParam(query, row, 1, DataType.NUMERIC, false);
            query = AddParam(query, row, 2, DataType.STRING, false);
            query = AddParam(query, row, 3, DataType.NUMERIC, false);
            query = AddParamPlain(query, "n", DataType.STRING, true);

            return query;
        }
    }
}

File: Backend/MyDreamHouse2/app/models/hauspaket_attribut_wert.rb
class Hauspaket_attribut_wert
	attr_reader :wert_id,:attribut_id,:wert_text,:wert_ordnung,:archived
	def initialize(wert_id,attribut_id,wert_text,wert_ordnung,archived)

		@:wert_id=wert_id
		@:attribut_id=attribut_id
		@:wert_text=wert_text
		@:wert_ordnung=wert_ordnung
		@:archived=archived
		
	end
end